app/src/main/java/com/worldsnas/starplayer/model/LocalMusicProviderImpl.kt

```diff
@@ -43,22 +44,18 @@ constructor(private val contentResolver: ContentResolver) : LocalMusicProvider {
                     val album = cursor.getString(albumColumn)
                     val artist = cursor.getString(artistColumn)
 
-                    val contentUri =
-                        ContentUris.withAppendedId(uri, id.toLong()).path
-                    if (contentUri.isNullOrEmpty()) {
```

Comment:
in if mohem boda nabod?

Reply:
hichvaght true nmishod, hich musici ham nist ke address nadashte bashe